syntax = "proto3";

import "trade/vo/FundJourVo.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.trade.response";
option java_outer_classname = "TradeQueryHistoryFundJourListResponseProto";
message TradeQueryHistoryFundJourListResponse {
	repeated FundJourVo data = 1;
}
